api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -169,7 +172,7 @@
   private static DbUser dbUser;
   private static List<DbAccessModule> accessModules;
 
-  private int DUA_VERSION;
+  private int DUCC_VERSION;
```

Comment:
we will be renaming all of these soon: https://precisionmedicineinitiative.atlassian.net/browse/RW-4838